tests/rt.conf

```diff
@@ -80,7 +80,7 @@ RUN     | fv3_gfs_v16_restart
 RUN     | fv3_gfs_v16_stochy                                                                                                      |                                         | fv3 |
 RUN     | fv3_gfs_v16_RRTMGP                                                                                                      |                                         | fv3 |
 RUN     | fv3_gfs_v16_RRTMGP_c192L127                                                                                             |                                         | fv3 |
-RUN     | fv3_gfs_v16_RRTMGP_2thrd                                                                                                |                                         | fv3 |
+RUN     | fv3_gfs_v16_RRTMGP_2thrd                                                                                                |                                         | fv3 | fv3_gfs_v16_RRTMGP
```

Comment:
This test (fv3_gfs_v16_RRTMGP_2thrd) should not depend on fv3_gfs_v16_RRTMGP.

Reply:
It should also have the `fv3` entry removed, see https://github.com/ufs-community/ufs-weather-model/issues/525

Reply:
I tried to fix the issue [#525](https://github.com/ufs-community/ufs-weather-model/issues/525).  Just realized the threading test is actually compared with baseline control, not the control running with threading. I will remove the fv3 in fv3_gfs_v16_RRTMGP_2thrd instead